src/Models/Magnetization_transfer/mt_sat.m

```diff
@@ -107,34 +104,39 @@
 
             B1params = obj.options.B1correctionfactor;
 
-            [FitResult.MTSAT, R1] = MTSAT_exec(data, MTparams, PDparams, T1params, B1params);
-            FitResult.T1 = 1./R1;
+            [MTSAT, R1, R1cor, MTsatcor] = MTSAT_exec(data, MTparams, PDparams, T1params, B1params);
```

Comment:
The test suite currently fails due to the change on this line. Two extra outputs were set (R1cor and MTsatcor), but the function that's being called never defined these extra outputs and still only outputs 2 values: https://github.com/qMRLab/qMRLab/blob/21bee21d2bd96176d3e450e48692e6d50b5586e7/src/Models_Functions/MTSATfun/MTSAT_exec.m#L3

Despite an attempt at a workaround below on lines 119 and 123, the `~isempty` call never get set since the code errors here.

It's not clear to me what exactly the two extra variables are expected to be (I could imagine that MTsatcor would maybe be with B1 correction from the code, but not sure about R1cor), I'll instead just add some placeholder "empty" variables that get outputted, and we can touch on this during the full review later on.

Reply:
Handled in [dc2f259](https://github.com/qMRLab/qMRLab/pull/484/commits/dc2f25943a0c6850604d614d477297c553b4a61c), but will need to revisit and/or check with Agah later what the intended behaviour was while this was being developped.